charts/alertmanager/values.yaml

```diff
@@ -357,18 +477,21 @@ templates: {}
 #   alertmanager.tmpl: |-
 
 ## Optionally specify extra list of additional volumeMounts
-extraVolumeMounts: []
+extraVolumeMounts:
+  []
   # - name: extras
   #   mountPath: /usr/share/extras
   #   readOnly: true
 
 ## Optionally specify extra list of additional volumes
-extraVolumes: []
+extraVolumes:
+  []
   # - name: extras
   #   emptyDir: {}
 
 ## Optionally specify extra environment variables to add to alertmanager container
-extraEnv: []
+extraEnv:
```

Comment:
please restore old formatting...

Reply:
sorry about the mess with the formatting - one of my code formatters was running haywire and I didn't notice. I'll revert those.